pegasus/sites.v3/code.org/public/music/index.haml

```diff
@@ -0,0 +1,88 @@
+---
+title: Music Lab
+theme: responsive_full_width
+---
+
+- button_url = CDO.studio_url("/s/music-intro-2024")
+- button_string = hoc_s("music_lab.button.try_music_lab")
```

Comment:
just out of curiousity, why are these variables as opposed to calling these utilities inline?

Reply:
This button will be used in multiple sections, but haven't added them yet. They'll be in the followup tickets.